e2e/cypress/support/commands.js

```diff
@@ -101,15 +122,15 @@ Cypress.Commands.add('ensureUser', (userLabel = 'participant') => {
     () => {
       cy.fixture('users').then((usersJson) => {
         const user = usersJson[userLabel]
-        cy.register(user)
+        cy.login(user)
       })
     },
-    {
-      validate: () => {
-        cy.getCookie('token2').should('exist')
-        cy.getCookie('uid2').should('exist')
-      },
-    },
+    // {
+    //   validate: () => {
+    //     cy.getCookie('token2').should('exist')
+    //     cy.getCookie('uid2').should('exist')
+    //   },
+    // },
```

Comment:
these cookies won't exist until after the user has logged in AND hit an endpoint requiring auth